components/playlist/browser/BUILD.gn

```diff
@@ -1,25 +1,14 @@
 # Copyright (c) 2021 The Brave Authors. All rights reserved.
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
-# You can obtain one at http://mozilla.org/MPL/2.0/.
+# You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import("//brave/build/config.gni")
-import("//brave/components/playlist/buildflags/buildflags.gni")
+import("//brave/components/playlist/common/buildflags/buildflags.gni")
 
-source_set("playlist_features") {
-  assert(enable_playlist)
-
-  sources = [
-    "features.cc",
-    "features.h",
-  ]
-
-  deps = [ "//base" ]
-}
-
-source_set("playlist") {
-  assert(enable_playlist)
+assert(enable_playlist)
 
+source_set("browser") {
```

Comment:
please change to `static_library` and we should add a follow-up issue to change to `component`

Reply:
Good call. I was wondering why we don't use component target whe I joined the team. filed an issue here: https://github.com/brave/brave-browser/issues/27764